docs/learn/getting-started-boss-room.md

```diff
@@ -92,11 +92,11 @@ To run multiple instances of the same app, you need to use the command line:
 In contrast to running a local setup, when playing over internet we do not necessarily need a built executable. You can run the game in editor.
 :::
 
-Running the game over internet currently requires either setting up a [Photon Transport for MLAPI](#photon-transport) or using [Port Forwarding](#port-forwarding).
+Running the game over internet currently requires either setting up a [Photon Transport for Netcode](#photon-transport) or using [Port Forwarding](#port-forwarding).
 
 #### Photon Transport
 
-[Photon Transport for MLAPI](https://github.com/Unity-Technologies/mlapi-community-contributions/tree/master/Transports/com.mlapi.contrib.transport.photon-realtime) uses Photon relay server to facilitate communication between clients and server living on different networks.
+[Photon Transport for Netcode](https://github.com/Unity-Technologies/mlapi-community-contributions/tree/master/Transports/com.mlapi.contrib.transport.photon-realtime) uses Photon relay server to facilitate communication between clients and server living on different networks.
```

Comment:
IIRC, this thing is now called as "Photon Transport for Netcode for GameObjects" @LukeStampfli